frontend/public/locales/en/translation.json

```
@@ -2530,6 +2538,7 @@
   "Subscription": "Subscription",
   "Subscription deployed on local cluster": "Subscription deployed on local cluster",
   "Subscription ID": "Subscription ID",
+  "Subscription name should be correct one": "Subscription name should be correct one",
```

Are we providing instructions to the user? Are we telling the user to just "Use the correct subscription name".
```suggestion
  "Be sure to use the correct Subscription name": "Be sure to use the correct Subscription name ",
```

---

These Helper texts are not showing so I removed all helper text